modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/ConfigurationContainer.tsx

```diff
@@ -102,6 +106,19 @@ export function ConfigurationContainer({
 					}
 					toggled={values.enableComments}
 				/>
+
+				<ClayTooltipProvider>
+					<span
+						title={Liferay.Language.get(
+							'you-can-manage-comments-in-the-page-builder-and-the-headless-api'
+						)}
+					>
+						<ClayIcon
+							className="lfr-objects__comments-tooltip-icon"
+							symbol="question-circle-full"
+						/>
+					</span>
+				</ClayTooltipProvider>
```

Comment:
Where is the FF?
See the code difference in here > https://github.com/brianchandotcom/liferay-portal/pull/169014/changes/da2a1daec1ffb5c086c864786f757064af2c675e

review the FF LPD number